src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -371,10 +371,10 @@ Item {
                 userName: model.name
                 chatKey: root.compressedKeyGetter(model.publicId)
                 trustIndicator: model.trustIndicator
-                isMutualContact: model.isMutualContact
-                image.source: model.icon
-                image.isIdenticon: model.isIdenticon
-                isOnline: model.onlineStatus
+                isMutualContact: !!model.isMutualContact
+                image.source: !!model.icon ? model.icon : ""
+                image.isIdenticon: !!model.isIdenticon
+                isOnline: ! model.onlineStatus
```

Comment:
do you need a space or you missed adding a second ! here?

Reply:
Oh, I'm embarrassed :)